src/content/overview/architecture/_index.md

```diff
@@ -11,7 +11,77 @@ user_questions:
+
+## Observability
+
+Observability on the Giant Swarm platform is based in Mimir, Loki and Grafana. The stack runs in the management cluster and provides a centralized view of the platform metrics, logs, and traces. You can add application metrics and logs letting the your teams detect anomalies, troubleshoot issues, and optimize resources efficiently. The platform supports customizable dashboards and alerts, enabling proactive management and a deeper understanding of application behavior and infrastructure status.
+
+## Single sign-On
+
+Single Sign-On (SSO) simplifies user authentication across the Giant Swarm platform by allowing users to access multiple services with your existing identity provider. It enhances security and user convenience by reducing password fatigue and centralizing identity management. You can build on top of `Kubernetes` Role-Based Access Control (RBAC) to define fine-grained access policies and permissions, ensuring that only authorized users can interact with the platform resources.
+
+## Network and connectivity
+
+The different technologies used in the Giant Swarm platform provides you with a secure and reliable network and connectivity between your applications and services. Platform engineers can create network policies to control traffic flow and enforce security rules, ensuring that data is protected and isolated. Relying on the built-in networking capabilities of `Kubernetes`, the platform supports many routing and load balancing strategies, enabling seamless communication between services and applications.
+
+## Runtime security
+
+Runtime security focuses on protecting applications and infrastructure during execution. [Falco](https://www.cncf.io/projects/falco/) project provide you with intrusion detection, vulnerability scanning, and automated threat response capabilities. The Giant Swarm platform ensures that security policies are enforced consistently, reducing the attack surface and mitigating risks.
+
+## Cost management
+
+Cost control is becoming increasingly important as organizations seek to optimize their cloud spending. Based on lessons learned from our experience with customers, the platform is built to help you manage costs effectively. Our solution engineers will help you to add visibility and control over your resource usage and expenses, leading to better decision-making and cost savings.
+
+## Automatic cluster management
+
+Leaning on the `Cluster API`, the Giant Swarm platform provides solid automation to manage the lifecycle of your clusters. On top, our team has developed additional logic cover certain aspects of the cluster lifecycle not included in the upstream project.
```

Comment:
```suggestion
Based on `Cluster API`, the Giant Swarm platform provides solid automation to manage the lifecycle of your clusters. On top of it, our team has developed additional logic to cover certain essential aspects of the cluster lifecycle not included in the upstream project.
```

Reply:
"Management" is such a vague word that one could replace it with "juggling" and not have a less informational sentence 😆.

Also I disagree with mentioning 'stuff not included upstream' in such a way. Without explaining what capabilities we offer like that, it's 1) an implementation detail and 2) screams "vendor lock-in".

I'm adding a suggestion below, covering both paragraphs – since the second paragraph greatly lists the capabilities.